### DISCUSSION TITLE
Is there a way to allow users to edit form blueprints, without giving them access to all blueprints?

### DISCUSSION THREAD

**Discussion by wiegertschouten** [statamic/cms#7910] [Q&A] [answered]
Right now, when I want to give users the permission to manage forms, they cannot edit the form's blueprints. I have to give them permission to edit blueprints in order to allow for that. Which is not what I want, because they can then edit *all* blueprints. Is there a way around this?

**Comment by duncanmcclean** [comment, marked as answer]
Sadly, this isn't possible at the moment. 

However, feel free to [add your 👍  to the open feature request](https://github.com/statamic/ideas/issues/531).

**Reply by wiegertschouten** [reply]
Thanks for pointing me there!
